library/src/lib/list/list-item.component.ts

```diff
@@ -9,5 +21,6 @@ import { Component, HostBinding } from '@angular/core';
     `
 })
 export class ListItemComponent {
+    /** @hidden */
     @HostBinding('class.fd-list-group__item') true;
```

Comment:
can you please make this 

@HostBinding('class.fd-list-group__item')
fdListItemClass: boolean = true;